yw_website/yw_website/apps/website/templates/extendable/_base_detailed_workflow.html

```diff
@@ -36,43 +58,38 @@
                 <div class="dropdown-content">
                     {% for version_ in versions %}
                         {% if version_.id != version.id %}
-                            <a href="{% url 'detailed_workflow' workflow.id version_.id %}">Version #{{ forloop.counter }}</a>
+                            <a href="{% url 'detailed_workflow' workflow.id version.id %}">Version #{{ forloop.counter }}</a>
```

Comment:
should be version_ instead of version as version is the current version of the page, whilst version_ is the version loop variable